index.ts

```diff
@@ -108,42 +118,43 @@ app.post('/api/psychologue/sendMail',
 app.post('/api/psychologue/login', speedLimiterLogin, loginController.login);
 app.get('/api/connecteduser', speedLimiter, loginController.connectedUser);
 
-app.get('/api/appointments', appointmentsController.getAppointments);
-app.post('/api/appointments',
+app.get('/api/psychologue/logout', speedLimiter, loginController.deleteToken);
+
+app.get('/api/appointments', xsrfProtection, appointmentsController.getAppointments);
```

Comment:
Can we apply xsrfProtection middleware for all `/api/*` endpoints and use `unless` to list the one where it doesn't apply (as it is done for `expressJWT`)?
I think it's better to apply it by default to avoid adding a new unprotected endpoint in the future (the existing test will still pass if we forgot to add the endpoint in it too).

Reply:
Also, there are still some warnings about csrf. A way to remove these and to handle all endpoints as noted before :point_up_2:  can be to use [csurf](https://github.com/expressjs/csurf) library. What do you think?